luxonis_ml/data/utils/data_utils.py

```diff
@@ -27,3 +28,44 @@ def _check_valid_array(path: str) -> bool:
             )
         if not _check_valid_array(value):
             raise Exception(f"Array at path {value} is not a valid numpy array (.npy)")
+
+
+def validate_text_value(
+    value: str,
+    classes: List[str]
+) -> Tuple[str, int]:
+    """Validates a text value to only contain valid classes.
+
+    @param classes: valid character classes.
+    @type classes:
+    @type value: str
+    @param value: text value to validate.
+    @rtype: str
+    @return: same input value if it's valid, raises a Warning and cleans/ignores invalid characters.
+    """
+    clean_value = ""
+    text_value, max_len = value
```

Comment:
Is it guaranteed the `value` will always contain exactly 2 characters? Even if so, maybe going with `Tuple[str, str]` would be better. In the very least documenting the 2-char requirement.